### ISSUE TITLE
deno.jsonc ignored?

### ISSUE THREAD

**Issue by JustinGrote** [skymethod/denoflare#30]
Trying to use import maps to replace my deps.ts file. 
https://deno.land/manual@v1.12.2/npm_nodejs/import_maps

Works just fine with deno run, however when I use denoflare I get an `export module undefined`. It appears it is ignoring my `deno.jsonc`?

**Comment by johnspurlock-skymethod** [comment]
yea you are in uncharted territory there - are you wanting to use in `denoflare serve`, `denoflare push`, both?

Shouldn't be too difficult, just would need to config or pass the path to deno.jsonc (or mimic deno's automagic which I'm not necessarily too keen on)

**Comment by johnspurlock-skymethod** [comment]
@JustinGrote mind sending me or posting an example project including your import maps?

**Comment by JustinGrote** [comment]
> @JustinGrote mind sending me or posting an example project including your import maps?

I actually bailed on Denoflare and went back to Wrangler and node. It's the standard import map examples I linked in previous post.

**Comment by johnspurlock-skymethod** [comment]
Alright, will reopen if there are more people blocked on it. Personally I still use deps.ts

**Comment by JustinGrote** [comment]
@johnspurlock-skymethod wanted to revisit this now that I'm trying to come back to Denoflare. Since you generate your own deno config file when you run deno bundle, if there was a way to inject additional deno command line options like `--import-map`, that would solve my problem. Alternatively allow a way to provide a deno.json file that would be merged with yours, or support import-map option as part of `.denoflare` config.

Basically I just need a way to pass --import-map to denoflare bundle, whatever that looks like, whenever a serve or push happens.

**Comment by johnspurlock-skymethod** [comment]
There are two "backends" when doing internal bundling in Denoflare, since the excellent `Deno.emit` API in earlier Deno versions was abruptly removed and replaced with a not quite ready to use external module which does not (and probably will not) ever support the same features.  See the gory details in Denoflare's `bundle.ts`.

So when running on recent Deno versions, I actually call `deno bundle` under the hood by default (you can still try out the other backends using a command line option) - not ideal, since it spawns another process and therefore needs exec permission.

All that to say I'd like to keep the option open to returning to an in-process bundle, if the external module ever reaches parity - so I want to be conservative about adding too many new compiler options etc that wouldn't be available outside of `deno bundle`.

I'd be fine with adding an --import-map to `serve/push` - unless you can think of specific other features in the deno.json file that are crucial to carry over as well.

**Comment by thebergamo** [comment]
I think the main case here would be to whenever we're running denoflare and we find a deno.json{c} it should be taken into consideration. 
importMaps is one of the changes I had to make in my project to make it work, so I had to completely ignore it to make denoflare to work and re-wrote most of my imports to consume directly from the `deps.ts` file which is honestly not that convenient if you have a project already working.

**Comment by fnhipster** [comment]
This is a blocker for my project since it depends on import maps for externals. Like it was stated in the description of this issue, it works fine on `deno run` Has anyone found a workaround either with **denoflare** or **wrangler**?

**Comment by johnspurlock-skymethod** [comment]
👋 everyone, I'm pretty busy at the moment - but if folks want me to prioritize this, head over to [Support this project](https://denoflare.dev/#support-this-project) and donate to help fund development.
